livekit-android-sdk/src/main/java/io/livekit/android/room/track/LocalVideoTrackOptions.kt

```diff
@@ -54,6 +54,7 @@ constructor(
 
 data class VideoEncoding(
     val maxBitrate: Int,
+    val minBitrate: Int,
```

Comment:
Can you make this an optional and set the default value to null instead? I'm not sure whether the underlying libwebrtc uses a default of 0.